### PR TITLE
Update global-design-system.md

### PR DESCRIPTION
**Pull request by seneca**
Please review these changes just in case.

### CODE REVIEW

File: packages/nuejs.org/docs/global-design-system.md [hunk 1]
```diff
@@ -20,16 +20,16 @@ Nue lets you assemble rich web pages without touching a single line of code. Ins
...
 
 
 ### Extended Markdown flavor
 Nue comes with a [Markdown flavor](content.html) for rich, interactive content.
-Content authors and UX developers can build web pages without ever touching a single line of code. Things like sections, grid layouts, buttons, tables, responsive images, videos, tabbed content, and code blocks are built in.
+Content authors and UX developers can build web pages without ever touching a single line of code. Things like sections, grid layouts, buttons, tables, responsive images, videos, tabbed content or code blocks are built in.
 
-This content assembly format always produces the same HTML output making it a perfect fit for the global design system. The produced markup is 100% headless so you can give your content a wildly different look with CSS.
+This content assembly format consistently generates identical HTML output, making it a perfect fit for the global design system. The resulting markup is 100% headless, allowing you to transform the appearance of your content only with CSS.
```
Comment (nobkd) [inline packages/nuejs.org/docs/global-design-system.md]:
```suggestion
This content assembly format consistently generates identical HTML output, making it a perfect fit for the global design system. The resulting markup is 100% headless, allowing you to transform the appearance of your content with only CSS.
```

**Review by nobkd — CHANGES_REQUESTED** [review]

**Comment by seneca** [comment]
On re-reading this sentence I tend to agree with you here.

**Review by nobkd — APPROVED** [review]

**Comment by tipiirai** [comment]
Damn. This has been changed by us both and now has conflicts. Can you resolve? 🙏

**Comment by nobkd** [comment]
Please check, if I merged something incorrectly. Thanks

**Comment by tipiirai** [comment]
Awesome. Thanks for fixing the conflicts!

**Pull request merged**